### PR TITLE
Date is removed in Join requests for meetup location- #334

### LINKED ISSUE #334: Bug - Date is empty in Join requests for meetup location.
## Description
Date is empty in every join requests for meetup location as seen in the image
## Mocks
image

## Acceptance Criteria
Join requests for a meetup location doesn't contain any date field. So, for every join request the date shows empty. Remove date. 

### Update [Required]
- [ ] Meet the Acceptance Criteria.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour
First timer task

### PR DESCRIPTION
**Pull request by SETIADEEPANSHU**
### Description
Date is removed in Join requests for meetup location. Now  "**Show Join requests**" does not show **date** field.
Fixes #334 

### Type of Change:
**Delete irrelevant options.**
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Changes in User- Interface has been reflected . Date field has been removed as per issue demand.
pr1

### Checklist:
**Delete irrelevant options.**
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Nik-Khandelwal** [comment]
Pls modify the line
Fixes #[344]  to Fixes #344 
This would reference the PR to the issue it is fixing.

**Comment by SETIADEEPANSHU** [comment]
Thank you @prachi1210 , it was my first pull request , although simple.

File: systers_portal/templates/meetup/join_requests.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h2 class="ml15">Join Requests</h2>
             <th>#</th>
             <th>User</th>
             <th>Email address</th>
-            <th>Date</th>
+            <!--<th>Date</th>-->
```
Comment (sa7i) [inline systers_portal/templates/meetup/join_requests.html]: This line has to be deleted, not commented out. @SETIADEEPANSHU

File: systers_portal/templates/meetup/join_requests.html [hunk 2]
```diff
@@ -19,7 +19,7 @@ <h2 class="ml15">Join Requests</h2>
               <th>{{ forloop.counter }}</th>
               <td><a href="{{ join_request.user.get_absolute_url }}">{{ join_request.user }}</a></td>
               <td><a href="mailto:{{ join_request.user.email }}">{{ join_request.user.email }}</a></td>
-              <td>{{ join_request.date_created }}</td>
+              <!--<td>{{ join_request.date_created }}</td>-->
```
Comment (sa7i) [inline systers_portal/templates/meetup/join_requests.html]: Same here.

**Comment by sa7i** [comment]
This can be merged if you can squash the two commits into one with a meaningful commit message. 
Thanks for the work and congrats on your first PR :)

**Comment by SETIADEEPANSHU** [comment]
I have squashed the commits , what should be the combined commit msg - "Date is removed in Join requests for meetup location" , will it work and also anywhere in commit message do I have to mention - closes #334 @sa7i

**Comment by sa7i** [comment]
@SETIADEEPANSHU Yes, the commit message looks good. You can do `git rebase -i origin/develop develop` and squash the commits after - by changing `pick` into `squash` except the one you want to commit.

**Comment by SETIADEEPANSHU** [comment]
Can you help me delete these mistakenly made commits @sa7i

**Comment by sa7i** [comment]
A quick fix is to close this issue, create a new branch and create a commit, since it's a small change. 
If you'd like to learn how to squash commits,  you could go through the git rebase [documentation](https://help.github.com/articles/about-git-rebase/), experiment with it and try to squash it by yourself. Hope this helps.

**Pull request closed without merging**